src/test/java/teammates/test/driver/TestUrl.java

```diff
@@ -0,0 +1,25 @@
+package teammates.test.driver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import teammates.common.util.Url;
+
+/**
+ *TestUrl.
```

Comment:
Missing space. Also, `Utility methods for Url class used only in tests.` might be better?